includes/reports/overview/list-table/class-sensei-reports-overview-list-table-abstract.php

```diff
@@ -46,6 +46,13 @@ abstract class Sensei_Reports_Overview_List_Table_Abstract extends Sensei_List_T
 	 */
 	protected $data_provider;
 
+	/**
+	 * All the item ids.
+	 *
+	 * @var array
+	 */
+	protected $all_item_ids = [];
+
```

Comment:
Same here - it could be reverted.

Reply:
Hey I reverted it but as a change, I didn't use git revert since this change is upon the other change and it would be too big of a mess reverting multiple things a top of one another
[7b0a333ac8bdfe493fa3152cc7ce04f3ee7210d1](https://github.com/Automattic/sensei/pull/5091/commits/7b0a333ac8bdfe493fa3152cc7ce04f3ee7210d1)